src/chart/MultiChart.js

```diff
@@ -115,6 +115,19 @@
         this._chartTypeProperties = _;
         return this;
     };
+    
+    MultiChart.prototype.getChartDataType = function () {
+        var context = this;
+        var types = ["1D","2D","ND"];
+        for(var i=0;i<types.length;i++) {
+            var idx = "_"+types[i]+"ChartTypes";
+            for(var j=0;j<context[idx].length;j++) {
+                if(context[idx][j].id === context.chartType()){
+                    return types[i];
+                }
+            }
+        }
+    };
```

Comment:
1.  Knowing what data "family" a widget belongs to is a useful thing in general I suspect (I will open an issue for this).
2.  Double loops are always candidates for performance bottlenecks and should be avoided

At line 16 add something along the lines of:
```
newItem.chartFamily = this._1DChartType.indexOf(item) >= 0 ? "1D" : this._2DChartType.indexOf(item) >= 0 ? "2D" : this._NDChartType.indexOf(item) >= 0 ? "ND" : "???";
```
And then use that.